spec/models/status_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
 RSpec.describe Status, :type => :model do
-  it { should have_valid(:description).when('In The Zone', 'Ready To Pair', 'Open To Help') }
-  it { should_not have_valid(:description).when('', nil, 'sadflkjsdlfk') }
+  it { should have_valid(:description).when('In The Zone', 'Ready To Pair', 'Open To Help', 'Sleeping') }
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.